common/lib/capa/capa/responsetypes.py

```diff
@@ -1957,7 +1977,36 @@ def validate_answer(self, answer):
             return False
 
     def get_answers(self):
-        return {self.answer_id: self.correct_answer}
+        _ = self.capa_system.i18n.ugettext
+        # Example: "Answer: Answer_1 or Answer_2 or Answer_3".
+        separator = u' <b>{or_separator}</b> '.format(
+            # Translators: Separator used in NumericalResponse to display multiple answers.
+            or_separator=_('or')
+        )
+        return {self.answer_id: separator.join([self.correct_answer] + self.additional_answers)}
+
+    def set_cmap_msg(self, student_answers, new_cmap, hint_type, hint_index):
+        """
+        Sets feedback to correct hint node in correct map.
+
+        Arguments:
+            student_answers (dict): Dict containing student input.
+            new_cmap (dict): Dict containing correct map properties.
+            hint_type (str): Hint type, either `correcthint` or `additional_answer`
+            hint_index (int): Index of the hint node
+        """
+        # Note: using self.id here, not the more typical self.answer_id
```

Comment:
The comment would be more useful if you explained why you aren't using the more typical self.answer_id.